en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,50 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
```

Comment:
Yes but I think it will not replace a clear and short sentence: a doc where you have to follow multiple links to get a single point is not user friendly :/